docker-compose.yml

```diff
@@ -45,6 +45,12 @@ services:
       - SYSTEMTESTS_VARIATION=${SYSTEMTESTS_VARIATION:-DEFAULT}
     volumes:
       - ./${SYSTEMTESTS_LOG_FOLDER:-logs}/docker/weblog/logs/:/var/log/system-tests
+    security_opt:
```

Comment:
I'm not sure this is required. Locally it was. I'd have to run it in GHA to check what is going on there.